### PR TITLE
165412886 ft view notifications

### PR DESCRIPTION
**Pull request by MBenedicte**
#### What does this PR do?
add a functionality of marking or notification as seen

#### Description of Task to be completed?
Having the following:
-Get notification feature

#### How should this be manually tested?
After cloning the repo, run the command **npm run dev** , then open the url "http://localhost:5000/profile" 

#### What are the relevant pivotal tracker stories?
165412886
#### Screenshots (if appropriate)
Screen Shot 2019-07-26 at 12 47 06

### CODE REVIEW

File: src/components/Header/Notifications/Modal/Modal.js [hunk 1]
```diff
@@ -45,63 +51,80 @@ class Modal extends Component {
...
+                        <FontAwesomeIcon icon={faCheckDouble} size="2x" className="text-success" />
+                      </button>
+                    </span>
+                  </div>
+                ))}
+              <div />
+              {notifications !== undefined
+                && (notifications.length === 0 ? (
+                  <div>
+                    <h3 className="center">No new notifications</h3>
```
Comment (e-liyai) [inline src/components/Header/Notifications/Modal/Modal.js]: why are we passing new notification object

Comment (e-liyai) [reply src/components/Header/Notifications/Modal/Modal.js]: also we can abstract this

Comment (MBenedicte) [reply src/components/Header/Notifications/Modal/Modal.js]: resolved

Comment (e-liyai) [reply src/components/Header/Notifications/Modal/Modal.js]: how has this been resolved?

Comment (MBenedicte) [reply src/components/Header/Notifications/Modal/Modal.js]: I was using if statement twice yet I can combine them in one statement.

**Review by kagaramag — APPROVED** [review]

**Review by luc-tuyishime — APPROVED** [review]

File: src/components/Header/Notifications/Modal/Modal.js (hunk 1 above)
Comment (e-liyai) [reply src/components/Header/Notifications/Modal/Modal.js]: why are we passing a new notification object?

Comment (MBenedicte) [reply src/components/Header/Notifications/Modal/Modal.js]: If you mean the "new notifications" on line 75, that is not an object. It is a message that is displayed when there is no notification

**Pull request merged**